scalar-aggregator/aggregator.c

```diff
@@ -622,42 +692,72 @@ int rule_save_data(rule_t *rule, ur_template_t *tpl, const void *record)
 
 int main(int argc, char **argv)
 {
-   int ret;
+   int ret = TRAP_E_OK;
+   int ret_val = EXIT_FAILURE;
```

Comment:
Here I would add some comment about what these return values are for. Maybe:
int ret = TRAP_E_OK; //used for working with TRAP functions return values
int ret_val = EXIT_FAILURE; // returned on module exit